src/Task.cc

```diff
@@ -236,8 +236,9 @@ WaitStatus Task::kill() {
       * the exit event, we already reaped it from the ptrace perspective,
       * which implicitly detached.
       */
-    if (ptrace_if_alive(PTRACE_GETEVENTMSG, nullptr, &raw_status)) {
-      status = WaitStatus(raw_status);
+    unsigned long long_status = raw_status;
```

Comment:
Assigning it to `raw_status` is not necessary, right?

Reply:
From the man page PTRACE_GETEVENTMSG seems to be just writing to data,
so yes, this assignment seems unnecessary. I will remove it.